app/assets/javascripts/cascade/level/rotator.js

```diff
@@ -37,11 +37,13 @@ $(function () {
             if ( !isSliderMasthead ) { return; }
 
             var currentSubtitle = currentSlideSubtitle(slider);
+
+            // Always remove contents of div when switching slides
             $subheader.empty();
 
             // Remove contents of subheader div if no subtitle present
             if ( currentSubtitle != '' && currentSubtitle !== undefined ) {
-                $subheader.append("<hr /><h3>"+currentSubtitle+"</h3>");
+                $subheader.append("<hr /><h3>"+currentSubtitle.replace(/(\n)+/g, "<br/>")+"</h3>");
```

Comment:
Isn't this being done in Cascade? I'm worried we not establishing a consistent practice here: sometimes we do it in Cascade (backend), sometimes in frontend. 

Also, if we do decide to do it here, I'd prefer this was done on its own line:

    // This allows users to insert their own linebreaks in the subtitle.
    currentSubtitle = currentSubtitle.replace(/(\n)+/g, "<br/>")

(`replace` returns a string?)

Reply:
No, this has to be here in the Javascript. The change in subtitle is associated with a click event, and we don't use formats for that. The only one that can be set by the format is the very first slide subtitle, when the page first loads.

Reply:
Ah, I see. Can you note this in a code comment here?

Reply:
There's already a comment in the velocity format about where the subtitle gets set in the javascript but I can add one to the javascript